meta/saimetadatatypes.h

```diff
@@ -918,6 +974,53 @@ typedef struct _sai_attr_capability_metadata_t
 
 } sai_attr_capability_metadata_t;
 
+/**
+ * @brief Defines statistics metadata.
+ */
+typedef struct _sai_stat_metadata_t
```

Comment:
you added new fields, but there is no added checks in saisanitycheck.c to check if this data is populated correctly